siatest/testgroup.go

```diff
@@ -48,13 +50,15 @@ var (
 
 // NewGroup creates a group of TestNodes from node params. All the nodes will
 // be connected, synced and funded. Hosts nodes are also announced.
-func NewGroup(nodeParams ...node.NodeParams) (*TestGroup, error) {
+func NewGroup(groupDir string, nodeParams ...node.NodeParams) (*TestGroup, error) {
 	// Create and init group
```

Comment:
Now that groups have a dir, it might make more sense to make the new file function a method of the group.

Reply:
Uploading and Downloading are methods on the `TestNode` which doesn't know about its group. That means an upload function can't just implicitly create a file because it doesn't hold a reference to the group.